File: API/Models/member.cs
using System;

namespace api.Models;

public class Member
{
    public int Id { get; set; }
    public string Name { get; set; } = "";
    public string Email { get; set; } = "";

    public ICollection<Book> BorrowedBooks { get; set; } = new List<Book>();
}

File: client/src/models/member.ts
import { Book } from './book';

export interface Member {
  id: number;
  name: string;
  email: string;
  borrowedBooks: Book[];  // Array of borrowed books (full objects)
}

export interface CreateMember {
  name: string;
  email: string;
  borrowedBookId?: number; // optional
}
